bids_prov/spm_load_config.py

```diff
@@ -23,12 +23,12 @@
 def get_empty_graph(context_url=CONTEXT_URL):
     return {
         "@context": context_url,
-        "@id": "http://example.org/ds00000X",
-        "generatedAt": "2020-03-10T10:00:00",
+        "BIDSProvVersion": "1.0.0",  # TODO ?
```

Comment:
To avoid any confusions, can we use "dev" as a version until we have a proper release of BIDS-Prov?

Reply:
yes